kerastuner/tuners/bayesian.py

```diff
@@ -71,42 +71,48 @@ def __init__(self,
         self._seed_state = self.seed
         self._tried_so_far = set()
         self._max_collisions = 20
-        self._num_trials = 0
-        self._score = {}
-        self._values = {}
-        self._x = None
-        self._y = None
         self.gpr = gaussian_process.GaussianProcessRegressor(
-            kernel=gaussian_process.kernels.ConstantKernel(1.0),
             alpha=self.alpha)
 
     def _populate_space(self, trial_id):
         # Generate enough samples before training Gaussian process.
-        if self._num_trials < self.num_initial_points or len(self._score) < 2:
-            self._num_trials += 1
-            values = self._new_trial()
-            self._values[trial_id] = values
+        completed_trials = [t for t in self.trials.values()
+                            if t.status == "COMPLETED"]
```

Comment:
We use `'` as the string delimiter in the rest of the file

Reply:
Done